internal/provider/lxd/testing_test.go

```diff
@@ -96,16 +99,20 @@ func (s *BaseSuiteUnpatched) SetUpSuite(c *gc.C) {
 	s.BaseSuite.SetUpSuite(c)
 }
 
-func (s *BaseSuiteUnpatched) SetUpTest(c *gc.C) {
-	s.BaseSuite.SetUpTest(c)
+func (s *BaseSuiteUnpatched) SetupMocks(c *gc.C) *gomock.Controller {
```

Comment:
knit: Doesn't need to be exported

Reply:
It does because of the `lxd` vs `lxd_test` setup :face_exhaling: 

I didn't want to re-write every test!